src/App.jsx

```diff
@@ -1,8 +1,17 @@
-import React from "react";
-import "./App.css";
+import React, { useEffect, useState } from "react";
+import "src/assets/styles/styles.scss";
+import SplashScreen from "src/components/splashScreen/SplashScreen";
+import Body from "src/components/body/Body";
 
 function App() {
-  return <></>;
+  const [isLoading, setIsLoading] = useState(true);
+  useEffect(() => {
+    setTimeout(() => {
+      setIsLoading(false);
+    }, 1000);
+  }, []);
+
+  return <div className="App">{isLoading ? <SplashScreen /> : <Body />}</div>;
```

Comment:
Are we having any Formatter here? If not [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) will be recommended.

Reply:
@fza-wtag Already used prettier for "printWidth": 80